index.html

```diff
@@ -1910,6 +1910,80 @@
       </pre>
       <section>
         <h2>
+          Creating a <code>PaymentAddress</code>
+        </h2>
+        <p>
+          The steps to <dfn>create a payment address</dfn> are given by the
+          following algorithm.
+        </p>
+        <ol data-link-for="PaymentAddress">
+          <li>Let <var>address</var> be a new instance of
+          <a>PaymentAddress</a>.
+          </li>
+          <li>Set <var>address</var>.<a>[[\addressLine]]</a> to the result of
+          splitting the user-provided address line into a <a data-cite=
+          "!WEBIDL#dfn-frozen-array-type">frozen array</a>. If none was
+          provided, set it to the empty <a data-cite=
+          "!WEBIDL#dfn-frozen-array-type">frozen array</a>.
+            <aside class="note">
+              How to split an address line is locale dependent and beyond the
+              scope of this specification.
+            </aside>
+          </li>
+          <li>Set <var>address</var>.<a>[[\country]]</a> to the user-provided
+          <a>country</a> as an upper case [[!ISO3166]] alpha-2 code, or to the
```

Comment:
Should not be `<a>country</a>` as that links to a dictionary member.

Reply:
good catch.